### DISCUSSION TITLE
How can I use privileged ports?

### DISCUSSION THREAD

**Discussion by truh** [podman-desktop/podman-desktop#7650] [Q&A] [answered]
When trying to bind on to a small port number like port 80 I get this error message

```
Command Output: Error: rootlessport cannot expose privileged port 80, you can add 'net.ipv4.ip_unprivileged_port_start=80' to /etc/sysctl.conf (currently 1024), or choose a larger port number (>= 1024): listen tcp 0.0.0.0:80: bind: permission denied
```

When using podman natively on Linux this is a perfectly helpful error message. With podman desktop it's not a clear what I need to do.

I wasn't able to find anything about this in the docs about privileged ports or even just how I would get a remote shell to the podman machine.

**Comment by afbjorklund** [comment]
* https://github.com/containers/podman-desktop/issues/4493

`podman machine ssh`

**Reply by truh** [reply]
Is there a way to run custom provisioning scripts like with colima?

**Reply by afbjorklund** [reply, marked as answer]
You can use Podman Desktop with lima, if you need scripts... Otherwise I think there is Ignition, for the CoreOS VM?
